.github/workflows/clean_dockerhub_images.yml

```diff
@@ -0,0 +1,31 @@
+name: Docker Cleanup
+
+on:
+  schedule:
+    - cron: '0 12 * * 1'  # Runs every Monday at noon (UTC)
+  workflow_dispatch:
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
only for the testing purpose, remove before merging

Reply:
please remove it before merging.